app.rb

```diff
@@ -27,8 +27,9 @@
 						 :email => params[:email],
 						 :password => params[:password] )
 	if user.save
-    session[:id] = user.id
-		redirect "/profile/#{user.username}"
+    	session[:id] = user.id
+    	session[:username] = user.username
+		redirect '/profile'
```

Comment:
this change was made in commit f363ce87 in sinatraapp.calvin.3! I'll delete
this branch, it's outdated
On Mar 20, 2015 10:47 AM, "Luka Milekic" <notifications@github.com> wrote:

> In app.rb
> <https://github.com/lmilekic/bluejadetwitter/pull/21#discussion_r26844835>
> :
>
> > @@ -27,8 +27,9 @@
> >  						 :email => params[:email],
> >  						 :password => params[:password] )
> >  	if user.save
> > -    session[:id] = user.id
> > -		redirect "/profile/#{user.username}"
> > +    	session[:id] = user.id
> > +    	session[:username] = user.username
> > +		redirect '/profile'
>
> This needs to be /profile/username
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lmilekic/bluejadetwitter/pull/21/files#r26844835>.
>